src/yunohost/tools.py

```diff
@@ -565,3 +565,25 @@ def tools_diagnosis(auth, private=False):
         diagnosis['private']['domains'] = domain_list(auth)['domains']
 
     return diagnosis
+
+def tools_public_key():
+    key_path = '/root/.ssh/id_rsa_ynh'
```

Comment:
Agreed. I asked @zamentur  who said it would easier to use it with borg in `/root/.ssh` but considering that `BORG_RSH` option I can put it in /etc/yunohost/keys/